src/components/RouterHistoryContainer.js

```diff
@@ -32,18 +31,19 @@ const RouterHistoryContainer = createReactClass({
     const state = router.current ? router.current.state : undefined;
     const replace = router.current ? router.current.replace : undefined;
 
-    return (
-      <div>
-        <Router {...this.props} router={router}/>
-        <History
-          history={this.props.history}
-          url={url} state={state} replace={replace}
-          isWaiting={!!router.next}
-          onChange={this.onChangeAddress}
-        />
-      </div>
-    );
-  }
+    return [
```

Comment:
This is only compatible with React 16, right? So I think we have two choices:

1. Detect the version of React and use a `div` tag in both `RouterContainer` and `RouterHistoryContainer` if React version < 16.
2. Bump redux-router-kit to 1.0.0 and change package.json to be React >= 16.

I'm cool with either. If we go with 2, I'd put a note in the readme saying that 1.0.0 requires React 16.